assets/xml/objects/object_dnt.xml

```diff
@@ -1,91 +1,106 @@
-        <Limb Name="object_dnt_Standardlimb_00ABE0" Type="Standard" EnumName="OBJECT_DNT_LIMB_19" Offset="0xABE0" />
-        <Limb Name="object_dnt_Standardlimb_00ABEC" Type="Standard" EnumName="OBJECT_DNT_LIMB_1A" Offset="0xABEC" />
-        <Limb Name="object_dnt_Standardlimb_00ABF8" Type="Standard" EnumName="OBJECT_DNT_LIMB_1B" Offset="0xABF8" />
-        <Skeleton Name="object_dnt_Skel_00AC70" Type="Flex" LimbType="Standard" LimbNone="OBJECT_DNT_LIMB_NONE" LimbMax="OBJECT_DNT_LIMB_MAX" EnumName="ObjectDntLimb" Offset="0xAC70" />
-        <Animation Name="object_dnt_Anim_00B0B4" Offset="0xB0B4" /> <!-- Original name is "dnt_walk" -->
-        <Texture Name="object_dnt_Tex_00B0D0" OutName="tex_00B0D0" Format="rgba16" Width="16" Height="16" Offset="0xB0D0" />
+        <!-- Animations -->
+        <Animation Name="gBusinessScrubShockedPoundAnim" Offset="0x994" /> <!-- Original name is "dnt_banban" -->
+        <Animation Name="gBusinessScrubTakeOffHatAnim" Offset="0x12F4" /> <!-- Original name is "dnt_boyon" -->
+
+        <!-- Additional Eyes DLs -->
+        <DList Name="gBusinessScrubEyesWideDL" Offset="0x1350" />
+        <DList Name="gBusinessScrubEyesSquintDL" Offset="0x1420" />
+
+        <!-- Animations -->
+        <Animation Name="gBusinessScrubFlyAnim" Offset="0x1BC8" /> <!-- Original name is "dnt_fly" -->
+        <Animation Name="gBusinessScrubShockedShakeHeadAnim" Offset="0x1E2C" /> <!-- Original name is "dnt_iyaiya" -->
+        <Animation Name="gBusinessScrubShockedEndAnim" Offset="0x2268" /> <!-- Original name is "dnt_iyaiyaTOmuun" -->
+        <Animation Name="gBusinessScrubFlyEndAnim" Offset="0x2670" /> <!-- Original name is "dnt_kouka" -->
+        <Animation Name="gBusinessScrubBurrowAnim" Offset="0x29E8" /> <!-- Original name is "dnt_moguru" -->
+        <Animation Name="gBusinessScrubThinkAnim" Offset="0x2F08" /> <!-- Original name is "dnt_muun" -->
+        <Animation Name="gBusinessScrubShockedStartAnim" Offset="0x3438" /> <!-- Original name is "dnt_nooo" -->
+        <Animation Name="gBusinessScrubExcitedAnim" Offset="0x38CC" /> <!-- Original name is "dnt_onedari" -->
+        <Animation Name="gBusinessScrubExcitedEndAnim" Offset="0x3CC0" /> <!-- Original name is "dnt_onedariTOwait" -->
+        <TextureAnimation Name="gBusinessScrubEmptyTexAnim" Offset="0x3CD0" />
+        <Animation Name="gBusinessScrubFlyStartAnim" Offset="0x4700" /> <!-- Original name is "dnt_spoon" -->
+        <Animation Name="gBusinessScrubRiseUpAnim" Offset="0x4AA0" /> <!-- Original name is "dnt_tobidasu01" -->
+        <Animation Name="gBusinessScrubJumpAnim" Offset="0x4E38" /> <!-- Original name is "dnt_tobidasu02" -->
+        <Animation Name="gBusinessScrubIdleStandAnim" Offset="0x5488" /> <!-- Original name is "dnt_wait" -->
+        <Animation Name="gBusinessScrubBobAnim" Offset="0x577C" /> <!-- Original name is "dnt_wait02" --> <!-- used as the idle anim when half burrowed into a deku flower-->
```

Comment:
For these two, it feels like you could drop "Stand" from the first one and just call it `gBusinessScrubIdleAnim`. If the second one was called `gBusinessScrubIdleBobAnim`, I could see why you would want to make the distinction, but at the moment there's only one idle animation, so you don't need to further disambiguate it.

Alternatively, you could just drop the word "idle" entirely and just call it `gBusinessScrubStandAnim`. Or you could put "idle" in both of them and call the second one `gBusinessScrubIdleBobAnim`. Up to you which you prefer; my key point here is that I don't think you need "idle" *and* "stand" in the first one unless "idle" is also in the second one.

Reply:
Removing `Idle` seems fine. Though I think I prefer `Standing` over anim. I know there has been some movement to remove `ing` from animations, but since this isn't an anim of standing up, but the anim done while standing I think it makes sense to keep the `ing`.

Reply:
Yeah, in this case, I think it's fine. The "movement" you've talked about is mostly me getting mad that sometimes we use `ing` and sometimes we don't; I don't think it's been widely discussed or agreed upon, so you shouldn't feel like you *have* to dogmatically conform to it.

Reply:
Fwiw I agree with said movement to an extent. In particular leave `ing` if it is more about the animation during that state (I have seen a couple like that, like the above mentioned one), and no `ing` if the verb describes the animation itself (for example if this was about the business scrub standing up I would use `StandAnim`)